etc/generate-apidocs-from-tag.pl

```diff
@@ -1,175 +0,0 @@
-#!/usr/bin/env perl
-use v5.14;
-use strict;
-use warnings;
-use utf8;
-use open qw/:std :utf8/;
-
-use Cwd qw/getcwd realpath/;
-use File::Temp qw/tempdir/;
-use List::Util qw/first/;
-
-# The required Doxygen version.
-# The generated results are sensitive to the release version.
-our $doxygen_version_required = "1.12.0";
```

Comment:
I noticed a dangling reference to this variable in the docs, the "IMPORTANT" note in `releasing.md` should refer to the new shell variable instead.